meeting/meeting_july_6_2021.md

```diff
@@ -0,0 +1,85 @@
+# July 6th (7 PM CST)/6th (9 AM JST) Meeting 2021
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Tibo)**
+* **#Query(Lloyd)**
+* **#Query(Chris)**
+* **#Query(Antoine)**
+* **#Query(Jesse)**
+* **#Query(Matt)**
+* **#Query(Ben)**
+* **#Query(Ivan!)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+* rust-dlc
+* atomic finance
+* others?
+
+## Specification Writing
+
+**#Status_Update_Interrupt**
+
+* Multi-oracle support merged!
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/128
+* Hyperbola curve piece support merged!
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/144
+* If anyone is interested it would be nice to document loss-of-fund vectors
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/132
+* ECDSA Adaptor Signature Specification merged!
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/114
+* Draft Updating P2P message serialization to include sub-types, optional fields, tlv streams
+  * https://github.com/discreetlogcontracts/dlcspecs/pull/163
+  * **#Discussion**
+    * Every tiny thing had a big type associated with it and this will simplify things and make them more clear and extensible, current LN is too restrictive
+    * Open call for review!
+  * Also adds protocol versioning to offer message
+    * Anyone have new thoughts on this?
+    * **#Discussion**
+* Mutual Close Message
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/161
+  * Current protocol has no cooperative close
+    * Could send PSBT, but this isn't ideal
+  * This is for after a funding tx is confirmed
+    * CET locktime hasn't been reached
+  * Extra input to mutual close to avoid free option, and in the future we will introduce revocation mechanism
+* Multi-oracle support for oracles with different num_digits algorithm proposed
+  * https://github.com/discreetlogcontracts/dlcspecs/issues/168
+  * Feedback welcome
+  * Implementation coming soon
+* Time to start thinking about Taproot!
+  * **#Discussion**
+* Path to Lightning
+  * **#Discussion**
```

Comment:
A standard default rounding mode that all wallets implement as policy, I think @benthecarman was going to send a mailing list post about performance woes we have experienced. Would be curious to hear input from @matthewjablack too